conda_build/build.py

```diff
@@ -438,6 +454,99 @@ def create_info_files(m, files, config, prefix):
                   config.timeout)
 
 
+def get_short_path(m, target_file):
+    entry_point_script_names = get_entry_point_script_names(m.get_value('build/entry_points'))
+    if is_noarch_python(m):
+        if target_file.find("site-packages") > 0:
+            return target_file[target_file.find("site-packages"):]
+        elif target_file.startswith("bin") and (target_file not in entry_point_script_names):
+            return target_file.replace("bin", "python-scripts")
+        elif target_file.startswith("Scripts") and (target_file not in entry_point_script_names):
+            return target_file.replace("Scripts", "python-scripts")
+    elif m.get_value('build/noarch_python'):
+        return None
+    else:
+        return target_file
+
+
+def sha256_checksum(filename, buffersize=65536):
+    sha256 = hashlib.sha256()
+    with open(filename, 'rb') as f:
+        for block in iter(lambda: f.read(buffersize), b''):
+            sha256.update(block)
+    return sha256.hexdigest()
+
+
+def has_prefix(short_path, files_with_prefix):
```

Comment:
I have a separate pr renaming `short_path` to `path` https://github.com/conda/conda-build/pull/1533.  It only renames the keys in `files.json` and keeps the name `short_path` everywhere else in the code.